src/app/App.js

```diff
@@ -8,7 +8,7 @@ import {
 import Landing from './Landing';
```

Comment:
Hey, I don't think you meant to actually commit this to master. I believe the correct component here might still be `Dashboard`? 

What I like to do when I want to maintain some local convenience like this while submitting the correct work is to be more selective when I stage files using Git.

For example, if I made some random changes like you did in `App.js` but only really want to submit my brand-spankin-new API interface, I might run:

```
git add src/controller/api*
git commit -m "Some commit message"
git push
```

Notice I used the wildcard symbol `*` to add all changes within a specific folder. Just a little workflow tip!

Reply:
GOOD TIP! I'll do that when viewing the topic dash now so I'm not continuously having to remember to change that route back. 

For App.js we still need landing.  I ran git checkout origin/master -- src/app/App.js so it matches master accurately.